src/main/docs/guide/quickStart.adoc

```diff
@@ -1 +1,178 @@
-TODO
\ No newline at end of file
+= Getting Started
+
+To add support for JMS, introduce the following dependency to your project
+
+_Gradle:_
+
+[source]
+----
+compile 'io.microanut.configuration:micronaut-jms-core:1.0.0'
+----
+
+_Maven:_
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.micronaut.configuration</groupId>
+    <artifactId>micronaut-jms-core</artifactId>
+    <version>1.0.0</version>
+</dependency>
+----
+
+If you are using a supported message broker (currently only ActiveMQ)
+then you will need to add the following properties to your `application.yml`
+
+[source,yaml]
+----
+micronaut:
+  jms:
+    activemq:
+      enabled: true
+      connection-string: 'tcp://host:61616'
+----
+
+and the following to your `build.gradle` or `pom.xml`
+
+[source]
+----
+implementation 'io.microanut.configuration:micronaut-jms-activemq:1.0.0'
+----
+
+or
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.micronaut.configuration</groupId>
+    <artifactId>micronaut-jms-activemq</artifactId>
+    <version>1.0.0</version>
+</dependency>
+----
+
+If your message broker is not supported, then you must provide
+the `ConnectionFactory` manually to the `BeanContext`
+in the following manner:
+
+[source,java]
+----
+@Factory
+public class JMSConfig {
+    @JMSConnectionFactory("myConnectionFactory")
+    public ConnectionFactory connFactory() {
+        return new ActiveMqConnectionFactory("tcp://host:616161");
+    }
+}
```

Comment:
In general we try to ensure snippets in documentation are tested, we probably need to use asciidoc includes and also support Groovy/Kotlin

Reply:
In general it seems the documentation needs quite a bit of work still

Reply:
Agreed, still learning the full power of AsciiDoc so I'll check out the other Micronaut projects for best practices and how best to structure the documentation